src/App.js

```diff
@@ -1,22 +1,126 @@
 import React from 'react';
-import './App.css';
+import classNames from 'classnames';
+import './App.scss';
+import Pagination from './components/Pagination';
+
+const perPageList = [
+  3, 5, 10, 20,
+];
 
 class App extends React.Component {
   state = {
-    tabs: [
-      { title: 'Tab 1', content: 'Some text 1' },
-      { title: 'Tab 2', content: 'Some text 2' },
-      { title: 'Tab 3', content: 'Some text 3' },
-    ],
+    currentPage: 1,
+    withInfo: false,
+    changeView: false,
+    perPageSelect: {
+      isShow: false,
+      perPage: 5,
+    },
+  };
+
+  handlerChangeView = () => this.setState(({ changeView }) => ({
+    changeView: !changeView,
+  }));
+
+  handlerWithInfoChange = () => this.setState(({ withInfo }) => ({
+    withInfo: !withInfo,
+  }));
+
+  onPageChange = currentPage => this.setState({ currentPage });
+
+  handlerDropdownToggle = () => this.setState(({ perPageSelect }) => ({
```

Comment:
if the function body is larger than one line, open parentheses and move it on the new line